packages/@okta/vuepress-site/docs/guides/migrate-to-idx/index.md

```diff
@@ -0,0 +1,111 @@
+
+However if you have [modified the template](https://developer.okta.com/docs/guides/style-the-widget/style-okta-hosted/) in certain ways, such as to perform redirects or set cookies, these modificiations may not be compatible with IDX.
+
+For reference, here is the default template:
+
+```html
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="robots" content="none" />
+
+    <title>{{pageTitle}}</title>
+    {{{SignInWidgetResources}}}
+</head>
+<body>
+    <div class="login-bg-image" style="background-image: {{bgImageUrl}}"></div>
+    <div id="okta-login-container"></div>
+
+    <!--
+        "OktaUtil" defines a global OktaUtil object
+        that contains methods used to complete the Okta login flow.
+     -->
+    {{{OktaUtil}}}
+
+    <script type="text/javascript">
+        // "config" object contains default widget configuration
+        // with any custom overrides defined in your admin settings.
+        var config = OktaUtil.getSignInWidgetConfig();
+
+        // Render the Okta Sign-In Widget
+        var oktaSignIn = new OktaSignIn(config);
+        oktaSignIn.renderEl({ el: '#okta-login-container' },
+            OktaUtil.completeLogin,
+            function(error) {
+                // Logs errors that occur when configuring the widget.
+                // Remove or replace this with your own custom error handler.
+                console.log(error.message, error);
+            }
+        );
+    </script>
+</body>
+</html>
+```
+
+The `success` callback from `renderEl` is being handled by `OktaUtil.completeLogin`. This method will correctly handle the IDX authentication flow.
+
+### Embedded signin widget
+
+To enable IDX in an embedded widget, set the option `useInteractionCodeFlow` to `true` on the signin widget configuration. All other options will continue to function as normal.
+
+:warning: Your code may break if it is calling the `renderEl` method and has logic which expects `sessionToken` or `session.setCookieAndRedirect` on the response object.
+
+Instead of `renderEl`, we recommend calling the method `showSignInToGetTokens`. This method receives and returns tokens without any browser redirect.
+
+```javascript
+var signIn = new OktaSignIn(
+  {
+    baseUrl: 'https://{yourOktaDomain}',
+    clientId: '{{clientId of your OIDC app}}'
+    redirectUri: '{{redirectUri configured in OIDC app}}'
+    useInteractionCodeFlow: true
+    // other options...
+  }
+);
+
+signIn.showSignInToGetTokens({
+  // Assumes there is an empty element on the page with an id of 'osw-container'
+  el: '#osw-container'
+}).then(function(tokens) {
+  // Store tokens
```

Comment:
should also handle the redirect or remove the SIW UI here?

Reply:
Yes, I did add that